src/prpl.ts

```diff
@@ -170,7 +170,7 @@ function loadBuilds(root: string, config: ProjectConfig|undefined): Build[] {
     console.info(
         `Registered entrypoint "${build.entrypoint}" with capabilities ` +
         `[${requirements.join(',')}].`);
-    if (!fs.existsSync(build.entrypoint)) {
+    if (!fs.existsSync(path.join(root, build.entrypoint))) {
```

Comment:
It's relative to the server root. Added a note, and a TODO to clean this up. Going to try to use type branding to make this clearer, in a followup PR.